processing/src/test/java/org/apache/druid/query/timeseries/TimeSeriesUnionQueryRunnerTest.java

```diff
@@ -49,6 +50,10 @@
   private final QueryRunner runner;
   private final boolean descending;
 
+  static {
```

Comment:
@xvrl this test was consistently failing in both 0.20.2 and 0.21. Should we raise a PR upstream as well for this?

Reply:
the correct way to fix this is to have the test extend `NullHandlingTest` rather than manually initializing it.
And yes, we should raise a PR for this if this is not already fixed in master upstream.

Reply:
will update this PR accordingly and then raise one upstream as well

Reply:
Orthogonal to this, some tests in 0.21 were failing because of [GC limit issues](https://confluentinc.semaphoreci.com/jobs/0b9f8229-428d-4147-a40a-888820669726)
Once I bumped [this](https://github.com/confluentinc/druid/blob/0.19.0-confluent/.semaphore/semaphore.yml#L81) from 800m to 1g, the build passed. This indicates something is consuming more resources in 0.21 right?